### DISCUSSION TITLE
Sometimes, when I hit backspace-enter, it acts like I've also hit ctrl-a

### DISCUSSION THREAD

**Discussion by DrReddP** [SubtitleEdit/subtitleedit#5076] [General]
This was a persistent problem about a year ago, but after a couple of updates it went away, but now it's back.

If I have a line that is too long and needs a break, I will place the cursor at the beginning of the word that I want to start the second line, and I will quickly hit backspace then enter to delete the space, and insert a line break. The problem is, that sometimes when I do this, it acts like I've hit control-a, and it selects all of the text.

It doesn't do it every time, even if it does it on a particular line and then I go back and try to do it again on the same line, it doesn't do it, but I managed to get it on camera only once. I've tried four different keyboards so it's not that.

Video: http://imgur.com/a/TbTwzie

**Comment by darnn** [comment]
I think it's related to this, since it also happens when another line is added:
https://github.com/SubtitleEdit/subtitleedit/issues/3141
In 3.6.1 NEXT beta 11:
![se-select all](https://user-images.githubusercontent.com/19672444/120282600-f1bbe900-c2c2-11eb-8dcd-068781b851e7.gif)

It's happening at the point at which the textbox switches from not having a scrollbar to having one. It sort of flashes, which I assume means it's getting replaced somehow by the scrollbar kind, and that's when all the text gets selected. Hard to do anything about if it can only be reproduced on certain machines, though, I suppose. For the record I'm back on a Windows 7 computer now.

Also, for the sake of trying to reproduce it: my max. number of lines is set to 2, and I'm using shift+enter in that gif in order to force SE to insert a line break even though it's over the limit. And, the entire text getting selected, it only happens once until you move to a different subtitle. So, like, if I put that subtitle back the way it was, two lines, and repeat what I did, it will not select the text. But if I go back one subtitle and then forward one subtitle, and do everything the same way, it will select all the text again.

**Comment by niksedk** [comment]
I have not been able to re-produce this in my Win 10 system.

Note that SE can use two different text boxes - one is the default text box, and the other is the rich text box which support colors. See Options - Settings - Appearance:
![image](https://user-images.githubusercontent.com/1261331/120368521-7550f680-c312-11eb-8ac5-5e675b1bc748.png)
(Syntax coloring will make SE use the `rich text box`)

What do you use? Does it happen with both for you? @DrReddP: Are you also on Win 7?

**Reply by DrReddP** [reply]
Where do I find this appearance setting

![image](https://user-images.githubusercontent.com/82276985/120383993-9524f200-c2f3-11eb-95d9-0fb84931639d.png)

**Reply by niksedk** [reply]
> 
> 
> Where do I find this appearance setting
> 

See Options - Settings - Appearance

Do read this page carefully: https://www.nikse.dk/SubtitleEdit/Help

**Reply by DrReddP** [reply]
I just showed you in that screenshot, when I hit options then settings there's no tab for appearance.

**Reply by darnn** [reply]
You seem to be using a fairly old version. The latest beta is here:
https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.1/SubtitleEditBeta.zip

**Reply by DrReddP** [reply]
I'm really not interested in beta testing, thank you. I'm on version 3.5.18
Is there a newer full version?

**Reply by darnn** [reply]
https://github.com/SubtitleEdit/subtitleedit/releases/tag/3.6.1

**Reply by DrReddP** [reply]
Wait, is 3.6.1 beta or not?

**Reply by DrReddP** [reply]
I updated to 3.6.0 and now the font here is all huge, how do I put it back? There's still no "appearance" tab
![image](https://user-images.githubusercontent.com/82276985/120387064-6f99e780-c2f7-11eb-8c88-4daf1b42cf42.png)

**Reply by OmrSi** [reply]
3.6.1 if official.
3.6.1 NEXT is the beta.

**Reply by DrReddP** [reply]
The 3.6.1 zip you posted above is giving me this. The other versions never did this
![image](https://user-images.githubusercontent.com/82276985/120387312-bd165480-c2f7-11eb-8d86-0d6304508675.png)

**Reply by OmrSi** [reply]
The one called SubtitleEditBeta.zip is the beta.

**Reply by DrReddP** [reply]
1. Why is Windows defending against 3.6.1
2. How do I get my old font back in the edit box. "Use syntax coloring" didn't change it

**Reply by darnn** [reply]
3.6.1 is the latest full release, so first of all, you should use that one.  The portable version is here:
https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.1/SE361.zip

As for getting your old font back, open the previous version you were using, open settings and check what you selected in the font tab. Then, in 3.6.1, go to settings, appearance (you should now have it on the left, like in the post above), and change the settings to those you were using previously.

As for Windows Defender, all I can say is that mine wasn't doing that in Windows 10. It's a false positive, either way.

**Comment by darnn** [comment]
I use the default one. This doesn't appear to happen for me with the rich text one. The rich text one doesn't blink in and out when the scrollbar is added, either. I seem to recall there was some specific reason I was using the default one (beyond the fact that I don't need syntax coloring), though, so I'd love it if it didn't happen there either.

**Comment by DrReddP** [comment]
Ty
